### ISSUE TITLE
Digger Github Locking Issue

### ISSUE THREAD

**Issue by liam-witterick** [diggerhq/digger#1669]
Hello, 

There's currently a problem across all of our projects using the Digger Github app. 

Our PRs are unable to acquire the lock and claims that the last merged PR has the lock. 

We've tried `digger unlock` but the issue remains. Have there been any changes since last week that would affect the Digger Github app in this way?

Error:

```
❌ Failed perform lock action on project: develop-database failed to lock project: Project<project> locked by another PR https://github.com/<project>/pull/14 (failed to acquire lock <project>). The locking plan must be applied or discarded before future plans can execute
```

**Version: diggerhq/digger@v0.6.0**

**Comment by motatoes** [comment]
Hello are you using cloud or self hosted backend ? 

Try to upgrade to the latest version of both to see if it resolves 

If using self hosted you can try to empty the digger_locks table

If you are using cloud and none of this helped please post this issue in support channel of slack to further investigate

**Comment by brianwpaxton** [comment]
Hi @motatoes, as per Liam's message above. How do we join the Slack community?

**Comment by motatoes** [comment]
Hi @brianwpaxton @liam-witterick its on digger.dev landing page button "slack community" , let me know if any issues joining with that link!

**Comment by brianwpaxton** [comment]
Thanks. It will only let you join if you have a digger.dev or tuition.io email address.

**Comment by motatoes** [comment]
oh .. let me check!

**Comment by UtpalJayNadiger** [comment]
@liam-witterick, @brianwpaxton - the link is now fixed. Apologies.
You can join slack [here](https://bit.ly/diggercommunity)

**Comment by motatoes** [comment]
This issue should have been addressed in #1675 , please re-open if there are still issues!
